apps/spruce/src/hooks/useTableSort/index.ts

```diff
@@ -7,7 +7,8 @@ import { queryString } from "utils";
 const { getSortString } = queryString;
 
 interface Props {
-  sendAnalyticsEvents?: (sorter?: SortingState) => void;
+  sendAnalyticsEvents?: (sorter: SortingState) => void;
+  singleQueryParam?: boolean;
```

Comment:
Could you add a comment referencing the ticket that will implement singleQueryParam as the default for all tables? That way it's clear it should be removed